x/auth/ante.go

```diff
@@ -126,36 +129,36 @@ func getSignerAccs(ctx sdk.Context, am AccountKeeper, addrs []sdk.AccAddress) (a
 			return nil, sdk.ErrUnknownAddress(addrs[i].String()).Result()
 		}
 	}
+
 	return
 }
 
 func validateAccNumAndSequence(ctx sdk.Context, accs []Account, sigs []StdSignature) sdk.Result {
 	for i := 0; i < len(accs); i++ {
-		// On InitChain, make sure account number == 0
+		// on InitChain make sure account number == 0
 		if ctx.BlockHeight() == 0 && sigs[i].AccountNumber != 0 {
-			return sdk.ErrInvalidSequence(
+			return sdk.ErrInvalidAccountNumber(
```

Comment:
I don't really think this is worth its own error code since it only applies at genesis. I feel like we will end up with way too many error codes if genesis things get their own error type.

Reply:
I forgot. `ErrInvalidAccountNumber` is actually needed in two places. Here and the check below.

Reply:
So in this case we can just use `CodeInvalidAccountNumber` or have TWO new error codes instead of one.

Reply:
oh sounds good!